pkgs/tools/misc/fetch-scm/default.nix

```
@@ -0,0 +1,44 @@
+{ stdenv, lib, fetchFromGitHub, guile }:
+
+let
+  file = "fetch.scm";
+  pkg = "fetch-scm";
+in
+  stdenv.mkDerivation rec {
+    pname = pkg;
+    version = "v0.1.5";
+
+    src = fetchFromGitHub {
+      owner = "KikyTokamuro";
+      repo = file;
+      rev = version;
+      sha256 = "sha256-H89VCNAYnTwVEqyInATvLHIB7ioe2zvIGTAM2MUo7+g=";
+    };
+
+    dontBuild = true;
+    dontPatchShebangs = true;
+
+    buildInputs = [ guile ];
+
+    installPhase = ''
+      runHook preInstall
+      # strip shebang, because nix removes `--no-auto-compile`
+      tail -n +5 ${file} > ${pkg}
+      # replace with new one
+      shebang="#!${guile.outPath}/bin/guile \\
+      --no-auto-compile -e main -s
+      !#
+      "
+      echo "$shebang$(cat ${pkg})" > ${pkg}
+      install -Dm755 -t $out/bin ${pkg}
+      runHook postInstall
+    '';
+
+    meta = with lib; {
+      description = "System information fetcher written in GNU Guile Scheme";
+      homepage = "https://github.com/${src.owner}/${src.repo}";
```

either option should work
```suggestion
      homepage = "https://github.com/KikyTokamuro/fetch.scm";
```
```suggestion
      inherit (src.meta) homepage;
```